webapp/components/_new/features/MySomethingList/MySomethingList.spec.js

```diff
@@ -114,13 +133,45 @@ describe('MySomethingList.vue', () => {
           )
         })
 
-        it('calls delete', async () => {
+        it('calls delete by committing "modal/SET_OPEN"', async () => {
           const deleteButton = wrapper.find('.base-button[data-test="delete-button"]')
           deleteButton.trigger('click')
           await Vue.nextTick()
-          const expectedItem = expect.objectContaining({ id: 'id', dummy: 'dummy' })
-          expect(propsData.callbacks.delete).toHaveBeenCalledTimes(1)
-          expect(propsData.callbacks.delete).toHaveBeenCalledWith(expect.any(Object), expectedItem)
+          // const expectedItem = expect.objectContaining({ id: 'id', dummy: 'dummy' })
+          // expect(propsData.callbacks.delete).toHaveBeenCalledTimes(1)
+          // expect(propsData.callbacks.delete).toHaveBeenCalledWith(expect.any(Object), expectedItem)
+          const expectedModalData = expect.objectContaining({
+            name: 'confirm',
+            data: {
+              type: '',
+              resource: { id: '' },
+              modalData: {
+                titleIdent: 'delete-modal.title',
+                messageIdent: 'delete-modal.message',
+                messageParams: {
+                  name: 'dummy',
+                },
+                buttons: {
+                  confirm: {
+                    danger: true,
+                    icon: 'trash',
+                    textIdent: 'delete-modal.confirm-button',
+                    callback: expect.any(Function),
+                  },
+                  cancel: {
+                    icon: 'close',
+                    textIdent: 'actions.cancel',
+                    callback: expect.any(Function),
+                  },
+                },
+              },
+            },
+          })
+          expect(mutations['modal/SET_OPEN']).toHaveBeenCalledTimes(1)
+          expect(mutations['modal/SET_OPEN']).toHaveBeenCalledWith(
+            expect.any(Object),
+            expectedModalData,
```

Comment:
Place the whole object here. If the test fails, you have to look up for the variable